Makefile

```diff
@@ -12,7 +12,7 @@ lint:
 docs: .venv-docs
 	# clean the build dir before rebuilding
 	cd docs; rm -rf _build/
-	cd docs; ../.venv-docs/bin/sphinx-build -d _build/doctrees -b dirhtml . _build/dirhtml
+	cd docs; ../.venv-docs/bin/sphinx-build -W --keep-going -d _build/doctrees -b dirhtml . _build/dirhtml
 
-clean:
+clean-docs:
 	rm -rf .venv-docs/
```

Comment:
`make clean` is a fairly bog-standard Makefile target.  Why change that here?

Reply:
Only because I never noticed that this was an option until I was about to add a target to do the same thing. From the outside, I had assumed `clean` would hit more than just the docs, since it's the repo's top-level Makefile. But I'm cool with keeping it as-is, too. Could also add a new `clean` target that depends on `clean-docs`

Reply:
Your call on the specifics and target dependencies, but I do think we should keep the `clean` target.  For those of us slightly longer in the tooth, this is a super, super standard&nbsp;&mdash;&nbsp;and actively utilized&nbsp;&mdash;&nbsp;target in Makefiles.

Reply:
Done